tests/e2e/test_io_performance.py

```diff
@@ -35,22 +35,47 @@ def base_setup(self, request, interface_iterate, pod_factory):
 
     @pytest.mark.parametrize(
         argnames=[
-            "size", "io_direction", "jobs",
+            "size", "io_direction", "bs", "jobs",
             "runtime", "depth", "sheet_index"
         ],
         argvalues=[
             pytest.param(
-                *['1GB', 'rw', 1, 60, 4, 1],
+                *['1000GB', 'rw', '4k', 1, 1200, 4, 4],
                 marks=pytest.mark.polarion_id("OCS-676")
             ),
             pytest.param(
-                *['1GB', 'rw', 6, 60, 16, 2],
+                *['1000GB', 'rw', '4k', 4, 600, 8, 5],
                 marks=pytest.mark.polarion_id("OCS-677")
             ),
             pytest.param(
-                *['1GB', 'rw', 12, 60, 32, 3],
+                *['1000GB', 'rw', '4k', 8, 600, 8, 6],
                 marks=pytest.mark.polarion_id("OCS-678")
             ),
+            pytest.param(
+                *['1000GB', 'ro', '4k', 1, 1200, 8, 7]
```

Comment:
@vasukulkarni This will most probably fail as we timeout all commands by 600 seconds.

Reply:
I had some commits locally, new commit increases the FIO_TIMEOUT